extensions/sql-database-projects/src/test/datasource.test.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as should from 'should';
+/*import * as should from 'should';
```

Comment:
I'd suggest just using .skip on the describe rather than commenting stuff out. Let's you fine tune it more easily and looks nicer. 

`describe.skip(...)`

Reply:
(you can do this for individual tests too which makes it a LOT easier since you don't have to worry about cleaning up imports and the like as well as you add/remove tests from being ran)

Reply:
Done. Thanks!